solidity/contracts/NetworkSettings.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "./INetworkSettings.sol";
+import "./utility/Owned.sol";
+import "./utility/Utils.sol";
+
+/**
+ * @dev This contract maintains the network settings.
+ */
+contract NetworkSettings is INetworkSettings, Owned, Utils {
```

Comment:
@yudilevi @barakman When do we want to add it to the test deployment script and the client registry? In this PR or the PR which will be actually using it?

Reply:
Can do it in this PR.

Reply:
👍

Reply:
Second thought, it looks pointless (redundant) in this PR; will do it when it is actually needed (i.e., in the test-deployment script, in the registry-client contract and perhaps also in one or more tests).

Reply:
Alright